test_runner/src/main/kotlin/ftl/util/LogTableBuilder.kt

```diff
@@ -145,3 +157,11 @@ private fun DataWithSize.center() = String.format(
 )
 
 inline fun TableColumn.applyColorsUsing(mapper: (String) -> SystemOutColor) = copy(dataColor = data.map(mapper))
+
+// RESOLUTION column needs dedicated alignment logic and data pre-processing
```

Comment:
Is this a todo or something that has been achieved?

Reply:
It's not a todo, it's just a short info why we have additional method here